packages/runtime/src/interpreter/interpreter.ts

```diff
@@ -146,6 +153,30 @@ export class Interpreter {
     });
   }
 
+  /**
+   * Description: moves instruction index to "label", throws error if label not found
+   * @param label: branch label
+   * @param line: line number
+   */
+  jumpToLabel (label: string, line: number): void {
+    const toInstructionIndex = this.labelMap.get(label);
+    if (toInstructionIndex === undefined) {
+      throw new RuntimeError(RUNTIME_ERRORS.TEAL.LABEL_NOT_FOUND, {
+        label: label,
+        line: line
+      });
+    }
+    let currentIndex = toInstructionIndex;
+    // if next immediate op is also label, then keep continuing, otherwise return
```

Comment:
So if we have something like:
```teal
loop:
b1:
b2:
int 2
int 3
+ 
..
// jump to loop (10 times)
```
Then it will skip b1 & b2 and set `instructionIndex` to `int 2` ? Is this verified on node?

Reply:
let's add this case in tests